Applications/Tizen_native/CustomShortcut/src/main.c

```diff
@@ -0,0 +1,140 @@
+/**
+ * @file main.c
+ * @date 14 May 2020
+ * @brief TIZEN Native Example App main entry with NNTrainer/CAPI.
+ * @see  https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#include "main.h"
+#include "view.h"
+#include <tizen.h>
+
+static int widget_instance_create(widget_context_h context, bundle *content,
+                                  int w, int h, void *user_data) {
+  widget_instance_data_s *wid =
+    (widget_instance_data_s *)malloc(sizeof(widget_instance_data_s));
+  int status = WIDGET_ERROR_NONE;
+
+  if (content != NULL) {
+    /* Recover the previous status with the bundle object. */
+  }
+  widget_app_context_set_tag(context, wid);
+
+  /* Window */
+  status = view_create(context, w, h);
+  if (status != WIDGET_ERROR_NONE) {
+    dlog_print(DLOG_ERROR, LOG_TAG, "failed to get window. err = %d", status);
+    return WIDGET_ERROR_FAULT;
+  }
+
+  /* Label*/
+  wid->label = elm_label_add(wid->conform);
+  evas_object_resize(wid->label, w, h / 3);
+  evas_object_move(wid->label, w / 4, h / 3);
+  evas_object_show(wid->label);
+  elm_object_text_set(wid->label, "Hello widget!");
+
+  return WIDGET_ERROR_NONE;
+}
+
+static int widget_instance_destroy(widget_context_h context,
+                                   widget_app_destroy_type_e reason,
+                                   bundle *content, void *user_data) {
+  widget_instance_data_s *wid = NULL;
+  widget_app_context_get_tag(context, (void **)&wid);
```

Comment:
Please check the return value of `widget_app_context_get_tag ()`

Reply:
Thank you, I checked the official docs and handled it :)